.github/workflows/_rust.yml

```
@@ -64,6 +64,79 @@ jobs:
       - uses: ./.github/actions/setup-rust
       - run: cargo test --all-features ${{ matrix.packages }}
 
+  temp-build-windows-artifacts:
```

```suggestion
  # TODO: Remove when windows build works reliably in cd.yml
  # These `temp-` jobs serve as a sanity check to early exit if there's 
  temp-build-windows-artifacts:
```

Might be helpful to leave a note for anyone working in this file for the nearterm.